cpp/src/gandiva/precompiled/extended_math_ops_test.cc

```diff
@@ -120,6 +120,46 @@ TEST(TestExtendedMathOps, TestRoundDecimal) {
   VerifyFuzzyEquals(round_float64_int32((double)INT_MIN - 1, 0), (double)INT_MIN - 1);
 }
 
+TEST(TestExtendedMathOps, TestBRoundDecimal) {
+  EXPECT_DOUBLE_EQ(bround_float64(0.0), 0);
+  EXPECT_DOUBLE_EQ(bround_float64(2.5), 2);
+  EXPECT_DOUBLE_EQ(bround_float64(3.5), 4);
+  EXPECT_DOUBLE_EQ(bround_float64(-2.5), -2);
+  EXPECT_DOUBLE_EQ(bround_float64(-3.5), -4);
+  EXPECT_DOUBLE_EQ(bround_float64(1.4999999), 1);
+  EXPECT_EQ(std::signbit(bround_float64(0)), 0);
+  EXPECT_EQ(std::signbit(bround_float64(0)), 0);
+  EXPECT_DOUBLE_EQ(bround_float64_int32(8.25, 1), 8.2);
+  EXPECT_DOUBLE_EQ(bround_float64_int32(8.35, 1), 8.4);
+  EXPECT_DOUBLE_EQ(bround_float64_int32(6.225, 2), 6.22);
+  EXPECT_DOUBLE_EQ(bround_float64_int32(8.335, 2), 8.34);
+  EXPECT_DOUBLE_EQ(bround_float64_int32(-8.25, 1), -8.2);
+  EXPECT_DOUBLE_EQ(bround_float64_int32(-8.35, 1), -8.4);
+  EXPECT_DOUBLE_EQ(bround_float64_int32(-6.225, 2), -6.22);
+  EXPECT_DOUBLE_EQ(bround_float64_int32(-8.335, 2), -8.34);
+  EXPECT_DOUBLE_EQ(bround_float64_int32(-6.225, -2), -6.22);
+  EXPECT_DOUBLE_EQ(bround_float64_int32(8.335, -2), 8.34);
+
+  VerifyFuzzyEquals(bround_float64(2.5), 2);
```

Comment:
What is the difference between VerifyFuzzyEquals and EXPECT_DOUBLE_EQ?

Reply:
VerifyFuzzyEquals has a tolefrance to be almost equal, not exactly the same number. Should I remove VerifyFuzzyEquals test cases?